### ISSUE TITLE
Add markers & custom markers on Static API

### ISSUE THREAD

**Issue by FlorianBurgevin** [mapbox/mapbox-sdk-py#188]
Hello,

Today, with the Static API, we can only add features in geojson with the sdk.

Is it possible to add markers and custom markers like the documentation below
[Static API category marker](https://www.mapbox.com/api-documentation/#retrieve-a-static-map-from-a-style)

Like `pin-s+blue(lng-lat)` or `url-{myurl}(lng-lat)`

Thank you

**Comment by sgillies** [comment]
@FlorianBurgevin yes, I think new `Marker` and `CustomMarker` classes would be an important addition to the SDK.
